src/main/kotlin/no/nav/familie/baks/mottak/søknad/PdfService.kt

```diff
@@ -52,11 +51,7 @@ class PdfService(
                     },
             )
 
-        return if (unleashService.isEnabled(FeatureToggleConfig.BRUK_NY_DOKGEN_LØSNING)) {
-            familieDokumentPdfClient.lagPdf(path, barnetrygdSøknadMapForSpråk + ekstraFelterMap)
-        } else {
-            dokgenPdfClient.lagPdf(path, barnetrygdSøknadMapForSpråk + ekstraFelterMap)
```

Comment:
Kan også fjerne dokgenPdfClient og unleashService fra konstruktøren til PdfService nå som de ikke lenger brukes

Reply:
[`00c0d1f` (#1119)](https://github.com/navikt/familie-baks-mottak/pull/1119/commits/00c0d1ffb3063451d922462bc5798805975faec4)

Reply:
Ble fjernet en linje for mye i testen, tror jeg, ellers ser det bra ut 👍